tabular/src/autogluon/tabular/models/fastainn/hyperparameters/parameters.py

```diff
@@ -48,4 +49,6 @@ def get_param_regression_baseline():
 
 
 def get_param_quantile_baseline():
-    return get_param_multiclass_baseline()
+    params = get_param_regression_baseline()
+    params.update({'alpha': 0.01})
```

Comment:
A comment would be helpful here to (briefly) describe alpha

Reply:
yes, I add some descriptions